Tests/BootCamp.Chapter.Tests/DeckTest.cs

```diff
@@ -0,0 +1,187 @@
+﻿using BootCamp.Chapter.Gambling;
+using FluentAssertions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace BootCamp.Chapter.Tests
+{
+    public class DeckTest
+    {
+        [Fact]
+        public void Deck_Created_WithoutCards_Throws_ArgumentNullException()
```

Comment:
Maybe `Deck_Created_With_Null_Cards...` Without cards seems to describe something different than null. 

Probably just semantics